Sources/Core/Microsoft.StreamProcessing/Microsoft.StreamProcessing.csproj

```diff
@@ -1320,5 +1320,9 @@
       <DependentUpon>FlushPolicy.tt</DependentUpon>
     </None>
   </ItemGroup>
+
+<!--  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.StreamProcessing.Provider\Microsoft.StreamProcessing.Provider.csproj" />
+  </ItemGroup> -->
```

Comment:
Delete? Is this a placeholder? I would have expected the provider project to take a dependency on the main one, not the other way around?

Reply:
It's indeed a placeholder, and sadly, the direction is correct.